app/models/rules_of_origin/query.rb

```diff
@@ -1,13 +1,19 @@
+# frozen_string_literal: true
+
 module RulesOfOrigin
   class Query
+    HEADING_CHECKER = /\A\d{6}\z/
+    COUNTRY_CHECKER = /\A[A-Z]{2}\z/
```

Comment:
what if we pass strings like "XY" ?
I guess it's ok not to check the country at that detail level, just the format here 🤔

Reply:
Yeh - they'll just get no matches - intention is to just not act like everything is in the API request when its not - previously we just returned empty data which will potentially confuse the person accessing the API